### ISSUE TITLE
Error when using 'normal' method call

### ISSUE THREAD

**Issue by harry-73** [jamauro/offline#10]
I have the following in my code
```
export const markermodify = new ValidatedMethod({
  name: 'marker.modify',
// ...
run: async function ({ _id, title, Type, subType, description, images }) {
  
     await Markers.updateAsync( ...);
  }
});

```

Client side: 

```
 Meteor.callAsync('marker.modify', state)
      .then(() => {
// ...
})
```

When I am offline and call the `marker.modify` method, I have an entry into the indexedDB in 'methods':
![image](https://github.com/user-attachments/assets/7d95a455-cdcd-41fb-963b-52ab4af7704c)

In this entry, there is missing field like 'args' and when I go back online, I have an error.

I was expected to have nothing in the methods IndexedDB as I don't use jam:method or queueMethod.

**Comment by jamauro** [comment]
Interesting. I'll take a closer look soon. Out of curiosity, were any methods at any point queued? Can you repro after clearing indexedDB with the browser console tools and starting fresh?

**Comment by harry-73** [comment]
It is what I did: Delete the db, reload the page. No methods were queued before the test.

**Comment by jamauro** [comment]
Thanks. This has been fixed in `v0.2.3`

**Comment by harry-73** [comment]
I confirm.
